client/src/components/Header.js

```diff
@@ -108,15 +108,14 @@ export default ({ authLoading, onMenuClick, isAuthenticated, user }) => {
         <Link to="/create-organization-profile">Add Organisation</Link>
       </Menu.Item>
       <Menu.Divider />
-      {user
-        ? user.organizations.map((organization) => (
-            <Menu.Item>
-              <Link to={`/organization/${organization._id}`}>
-                {organization.name}
-              </Link>
-            </Menu.Item>
-          ))
-        : null}
+      {user?.organizations?.map((organization) => (
```

Comment:
Looks good to me, whats the difference between the two? the second one checks for organization as well as the user ?

Reply:
Yes. Basically if `user` is `undefined` OR `organizations` is `undefined` it won't do anything.

It's a [new ecmascript feature](https://github.com/tejacques/es-safe-navigation) so transpiled via webpack to something like `users && user.organizations && user.organizations.map`